tests/test_api_integration.py

```
@@ -0,0 +1,244 @@
+mock_config.patch_config_module()
+
+# Mock discord before importing any project modules
+import sys
+_discord_mock = MagicMock()
+_discord_mock.Entitlement = MagicMock()
+sys.modules["discord"] = _discord_mock
+sys.modules["discord.ext"] = MagicMock()
+sys.modules["discord.ext.commands"] = MagicMock()
+sys.modules["discord.app_commands"] = MagicMock()
+
+from api import (
```

Flake8 (7.3.0)

[error] 'api.set_bot' imported but unused

(F401)